src/main.test.js

```diff
@@ -54,35 +54,35 @@ test("Valid input with comment,middle of line", () => {
 test("Valid input with comment, start of line", () => {
   const testEnv =
     "#DATABASE_URL: email = test@mail.com\nDATABASE_URL: email = test@mail.com";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpecToHTML(testEnv)).resolves.toEqual(
     `<label for="env_spec_database_url">DATABASE_URL</label>\n` +
       `<input id="env_spec_database_url" name="database_url" type="email" value="test@mail.com" />\n`
   );
 });
 
 test("Invalid environmental variable : starts with number", () => {
   const testEnv = "1DATABASE_URL\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv)).resolves.toEqual("Error:Wrong Syntax");
 });
 
 test("Invalid environmental variable : contains non alphanumeric characters", () => {
   const testEnv = "DATABASE_URLαα\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv)).resolves.toEqual("Error:Wrong Syntax");
 });
 
 test("Invalid environmental variable : contains lowercase letters", () => {
   const testEnv = "database_url\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv)).resolves.toEqual("Error:Wrong Syntax");
 });
 
 test("Invalid type", () => {
   const testEnv = "\nADMIN_EMAIL: notgood";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpecToHTML(testEnv)).resolves.toEqual("Error:Wrong Syntax");
```

Comment:
We better test `envSpec.parse` here instead and we better use [`.rejects`](https://jestjs.io/docs/en/tutorial-async#rejects) since syntax errors make promises get rejected.